utils/mwa_metadb_utils.py

```diff
@@ -156,4 +192,58 @@ def obs_max_min(obs_id):
     obs_end = int(max(times))
     return obs_start, obs_end
 
-
+def write_obs_info(obsid):
+    """
+    Writes obs info to a file in the current direcory
+    """
+    data_dict = getmeta(service='obs', params={'obs_id':str(obsid)})
+    filename = str(obsid)+"_info.txt"
+    logger.info("Writing to file: {0}".format(filename))
+
+    channels = data_dict["rfstreams"]["0"]["frequencies"]
+    centre_freq = ( min(channels) + max(channels) ) / 2. * 1.28
+    array_phase = get_obs_array_phase(obsid)
+
+    f = open(filename, "w+")
+    f.write("-------------------------    Obs Info    --------------------------\n")
+    f.write("Obs Name:           {}\n".format(data_dict["obsname"]))
+    f.write("Creator:            {}\n".format(data_dict["rfstreams"]["0"]["creator"]))
+    f.write("Array phase:        {}\n".format(array_phase))
+    f.write("~FWHM (arcminute)   {:4.2f}\n".format(calc_ta_fwhm(centre_freq,
+                                                           array_phase=array_phase)*60.))
+    f.write("Start time:         {}\n".format(data_dict["starttime"]))
+    f.write("Stop time:          {}\n".format(data_dict["stoptime"]))
+    f.write("Duration (s):       {}\n".format(data_dict["stoptime"]-data_dict["starttime"]))
+    f.write("RA Pointing (deg):  {}\n".format(data_dict["metadata"]["ra_pointing"]))
+    f.write("DEC Pointing (deg): {}\n".format(data_dict["metadata"]["dec_pointing"]))
+    f.write("Channels:           {}\n".format(channels))
+    f.write("Centrefreq (MHz):   {}\n".format(centre_freq))
+    f.close()
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description="""Returns information on an input Obs ID""")
+    parser.add_argument("obsid", type=int, help="Input Observation ID")
+    parser.add_argument("-w", "--write", action="store_true", help="OPTIONAL - Use to write results to file.")
+    args = parser.parse_args()
+
+    if args.write == False:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comparison to False should be 'if cond is False:' or 'if not cond:'](https://app.codacy.com/app/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=4498135)